src/universal/modules/outcomeCard/containers/OutcomeCard/OutcomeCardContainer.js

```diff
@@ -156,10 +180,11 @@ class OutcomeCardContainer extends Component {
           hasDragStyles={hasDragStyles}
           isAgenda={isAgenda}
           isDragging={isDragging}
-          isEditing={isEditing}
+          isEditing={isEditing()}
```

Comment:
Just curious, why `isEditing()` vs. `!this.state.activeEditingClients.isEmpty()`? More expressive?